### ISSUE TITLE
Drone not starting (timeouts?)

### ISSUE THREAD

**Issue by xhochy** [conda-forge/conda-forge.github.io#954]
It seems that in a significant number of cases drone.io is not starting to build.

 * a whole PR isn't recognized by drone: https://github.com/conda-forge/jupyter_core-feedstock/pull/22
 * Some times the webhook to drone time outs and using the Github Settings UI to retrigger the hook fixes this: https://github.com/conda-forge/jupyter_core-feedstock/pull/24#issuecomment-573693799

**Comment by xhochy** [comment]
Saw this again today on the master of `astroid`, `libprotobuf` and `protobuf`. For https://github.com/conda-forge/boost-feedstock/pull/82 I'm currently unable to trigger Drone at all and in https://github.com/conda-forge/grpc-cpp-feedstock/pull/45 I cannot get Drone to restart altough I'm logged in using admin rights.

**Comment by scopatz** [comment]
Yeah, I am seeing this as well (on `pugixml`). I did notice that if I waited a day and pushed up a new commit, drone would trigger

**Comment by jakirkham** [comment]
@isuruf, do you happen to know who we should be talking to at Drone about this?

**Comment by jakirkham** [comment]
Based on discussion in the core meeting earlier, we are trying to determine next steps for solving or mitigating this issue as it has become a significant pain point. Most notably webhooks are failing. Someone mentioned 20 builds missed after looking through feedstocks just this morning. Further we are not seeing any kind of notification (other than users complaining) about these packages being missing.

It would be good if someone could follow-up with Drone on this issue. Additionally it would be good if we investigated alternatives (should they be needed). One alternative raised during the meeting is using Travis CI for these builds. Have raised issue ( https://github.com/conda-forge/conda-smithy/issues/1229 ) to investigate how hard this would be to do.

Please feel free to correct anything I may have misunderstood. Also thoughts on this issue are generally welcome 🙂

cc @conda-forge/arm-arch

**Comment by isuruf** [comment]
https://gitter.im/drone/drone?at=5e3ae75c58f02e349765261f

**Comment by xhochy** [comment]
In https://github.com/conda-forge/grpc-cpp-feedstock/pull/46 drone.io now return `HTTP 400 {"message":"Invalid or malformed webhook"}`.
